docs/runtimes/images.md

```diff
@@ -5,16 +5,60 @@ Every release of MLRun includes several images for different usages. The build a
 
 **In this section**
 - [Using images](#using-images)
-- [MLRun images and how to build them](#mlrun-images-and-how-to-build-them)
+- [MLRun images](#mlrun-images)
+- [Building MLRun images](#building-mlrun-images)
 - [MLRun images and external docker images](#mlrun-images-and-external-docker-images)
 
 ## Using images
 
 See {ref}`build-function-image`.
 
-## MLRun images and how to build them 
+## MLRun images  
 
-See [README](https://github.com/mlrun/mlrun/blob/development/dockerfiles/README.md).
+Every release of MLRun includes several images for different usages. All images are published to 
+[dockerhub](https://hub.docker.com/u/mlrun) and [quay.iohttps://quay.io/organization/mlrun](https://quay.io/organization/mlrun]).
+
+The images are:
+
+`mlrun/mlrun`: The most basic (and smallest) image, can be used for simple jobs,  including Open MPI. This is simply MLRun installed on top of a python image.
+`mlrun/mlrun-gpu`: The same as `mlrun/mlrun` but for GPUs, including Open MPI. 
+`mlrun/ml-base`: The image for file acquisition, compression, dask jobs, simple training jobs and other utilities. Like `lrun/mlrun` with the addition of Miniconda and other python packages.
+`mlrun/jupyter`: An image with Jupyter giving a playground to use MLRun in the open source. Built on top of jupyter/scipy-notebook, with the addition of MLRun and several demos and examples.
+`mlrun/mlrun-api`: The image used for running the MLRun API.
+`mlrun/mlrun-ui`: The image used for running the MLRun UI.
+
+## Building MLRun images
+
+To build all images, run this command from the root directory of the mlrun repository:
+
+`MLRUN_VERSION=X MLRUN_DOCKER_REPO=X MLRUN_DOCKER_REGISTRY=X make docker-images`
+
+Where:
+- MLRUN_VERSION this is used as the tag of the image and also as the version injected into the code (e.g. latest or 0.7.0 or 0.6.5-rc6, defaults to unstable)
+- MLRUN_DOCKER_REPO is the docker repository (defaults to mlrun)
+- MLRUN_DOCKER_REGISTRY is the docker registry (e.g. quay.io/, gcr.io/, defaults to empty (docker hub))
+
+For example, running `MLRUN_VERSION=x.y.z make docker-images` generates these images:
+- mlrun/mlrun-api:x.y.z
+- mlrun/mlrun:x.y.z
+- mlrun/mlrun-gpu:x.y.z
+- mlrun/jupyter:x.y.z
+- mlrun/ml-base:x.y.z
+- mlrun/ml-models:x.y.z
+
+You can also build only a specific image, for example, `make api` (builds only the api image)
+Or a set of images, for example, `make mlrun jupyter base` 
+
+The possible commands are:
+- `mlrun`
+- `mlrun-gpu`
+- `api`
+- `jupyter`
+- `base`
+- `models`
```

Comment:
remove

Reply:
@liranbg That means that this:
You can also build only a specific image, for example, `make api` (builds only the api image)
Or a set of images, for example, `make mlrun jupyter base` 
can only be replaced with
You can also build only a specific image, for example, `make mlrun` (builds only the api image)
Or a set of images, for example, `make mlrun mlrun-gpu`
But is there a use case for the 2nd example?

Reply:
implemented without 2nd line